index.js

```diff
@@ -1,7 +1,13 @@
 /** @format */
 
-import {AppRegistry} from "react-native"
+import {
+    AppRegistry
+} from "react-native"
 import App from "./App"
-import {name as appName} from "./app.json"
+import {
+    name as appName
+} from "./app.json"
```

Comment:
Are these import linebreaks intentional?

Reply:
They are not. Thats my ESLint rule remnants. :D Can be removed